locales/de/calendar.json

```diff
@@ -0,0 +1,48 @@
+{
+  "date": "Datum",
+  "time": "Uhrzeit",
+  "event": "Formel 1 Grand Prix Veranstaltungen",
+  "notice": {
+    "text": "Die ersten acht Rennen des revidierten Kalenders unten. Times are TBC along with additional races.",
+    "link": "Mehr lesen"
+  },
+  "badges": {
+    "tbc": "Unbestätigt",
+    "tickets": "Karten",
+    "canceled": "Abgesagt",
+    "next": "NÄCHSTER"
+  },
+  "schedule": {
+    "fp1": "Freies Training 1",
+    "fp2": "Freies Training 2",
+    "fp3": "Freies Training 3",
+    "qualifying": "Qualifikation",
```

Comment:
In context with F1 Qualifying is just taken as an easier word to say and i would just take the english word.